resource-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/resource/redis/RedisPersistenceTest.java

```diff
@@ -60,11 +60,11 @@
 @SpringBootTest(
         classes = {RedisPersistenceTest.class, ResourceApplication.class},
         webEnvironment = WebEnvironment.RANDOM_PORT,
-        properties = {"akka.discovery.config.services.kafka.from-config=false", "spring.data.redis.repositories.key-prefix=test:"}
+        properties = {"akka.discovery.config.services.kafka.from-config=false"}
 )
 @Import({KafkaTestConfiguration.class})
 @ContextConfiguration(initializers = {RedisInitialiser.class})
-@ActiveProfiles({"redis", "akka-test", "testcontainers"})
+@ActiveProfiles({"redis-test", "akka-test", "testcontainers"})
```

Comment:
Revert these changes, i believe this is left over from debugging

Reply:
Reverted